common/src/main/java/com/wynntils/models/worlds/bossbars/InfoBar.java

```diff
@@ -13,13 +13,13 @@
 import java.util.regex.Pattern;
 
 public class InfoBar extends TrackedBar {
-    // §cDetlas§4 [AVO]
+    // Test in InfoBar_TERRITORY_INFO_PATTERN
     private static final Pattern TERRITORY_INFO_PATTERN =
-            Pattern.compile("§.(?<territory>.+)§. \\[(?<tag>[A-Za-z]{3,4})\\]");
+            Pattern.compile("§[abc](?<territory>.+)§[234] \\[(?<tag>[A-Za-z]{3,4})\\]");
 
     // §7Lv. 92§f - §bKingdom Foxes§f - §762% XP
     private static final Pattern GUILD_INFO_PATTERN =
-            Pattern.compile("§7Lv. (?<level>\\d+)§f - §b(?<guild>.+)§f - §7(?<xp>\\d+)% XP");
+            Pattern.compile("§7Lv\\. (?<level>\\d+)§f - §b(?<guild>[a-zA-Z ]+)§f - §7(?<xp>\\d+)% XP");
```

Comment:
```suggestion
            Pattern.compile("§7Lv\\. (?<level>\\d+)§f - §b(?<guild>[a-zA-Z\\s]+)§f - §7(?<xp>\\d+)% XP");
```
Perhaps this is a bit cleaner.